module-2/backend/week_two.md

```diff
@@ -1,37 +1,56 @@
 ## Week Two - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON - YOU are a web developer!!!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON - YOU are a web developer!!!).
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+It is a DSL that allows you to treat tables within a SQL database as Ruby objects.
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+Class methods. A horse belongs_to a stable, and a stable has_many horses.
```

Comment:
i think this question is misleading - i'm more interested in how you use the methods these associations give you (we talked about this in class today). in your example with `horses` and `stables`. the methods that these associations give you are instance methods 👍